EKF/ekf.h

```diff
@@ -57,58 +57,95 @@ class Ekf : public EstimatorInterface
 	// should be called every time new data is pushed into the filter
 	bool update();
 
-	// gets the innovations of velocity and position measurements
-	// 0-2 vel, 3-5 pos
-	void get_vel_pos_innov(float vel_pos_innov[6]);
+	// gets the GPS innovations of velocity and position measurements
```

Comment:
All those comments are redundant as the names of the functions are clear. However, if the intent is to generate documentation, you would need to add more details